cohorts/load.py

```diff
@@ -80,6 +86,18 @@ def __init__(self,
         self.nonsynonymous_effect_cache_name = "cached-nonsynonymous-effects"
         self.neoantigen_cache_name = "cached-neoantigens"
 
+    def verify_survival(self):
```

Comment:
How does it not allow for that? (In fact, I had a previous assert that at least some patients have OS == PFS, but took it out since it won't always be true.)